src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -114,11 +114,22 @@ private AddPropertyTransform<CommonCompilerOptions>[] KeyFileTransforms
             "**/*.cs"
         };
 
+        private static bool IsPlainFileName(string fileName)
+        {
+            return !fileName.Contains('/') && !fileName.Contains('\\');
```

Comment:
Should we also check for any file in the directory tree? Meaning anything that doesn't start with ..\ since the SDK included filter is **/*.cs